src/main/java/net/javadiscord/javabot/systems/qotw/submissions/dao/QOTWSubmissionRepository.java

```diff
@@ -24,17 +24,18 @@ public class QOTWSubmissionRepository {
 	 * @throws SQLException If an error occurs.
 	 */
 	public void insert(QOTWSubmission submission) throws SQLException {
-		PreparedStatement stmt = con.prepareStatement("INSERT INTO qotw_submissions (thread_id, question_number, guild_id, author_id) VALUES (?, ?, ?, ?)",
+		try (PreparedStatement stmt = con.prepareStatement("INSERT INTO qotw_submissions (thread_id, question_number, guild_id, author_id) VALUES (?, ?, ?, ?)",
 				Statement.RETURN_GENERATED_KEYS
-		);
-		stmt.setLong(1, submission.getThreadId());
-		stmt.setInt(2, submission.getQuestionNumber());
-		stmt.setLong(3, submission.getGuildId());
-		stmt.setLong(4, submission.getAuthorId());
-		int rows = stmt.executeUpdate();
-		if (rows == 0) throw new SQLException("Submission was not inserted.");
-		stmt.close();
-		log.info("Inserted new QOTW-Submission: {}", submission);
+		)) {
+			stmt.setLong(1, submission.getThreadId());
+			stmt.setInt(2, submission.getQuestionNumber());
+			stmt.setLong(3, submission.getGuildId());
+			stmt.setLong(4, submission.getAuthorId());
+			int rows = stmt.executeUpdate();
+			if (rows == 0) throw new SQLException("Submission was not inserted.");
+			stmt.close();
```

Comment:
I think that should be removed as it is no longer needed because of the try-with-resources block.

Reply:
+1